source/faq.txt

```diff
@@ -17,6 +17,21 @@ This page provides answers to some frequently asked questions we have
 encountered. If you have additional questions please contact MongoDB
 Support.
 
+Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
+---------------------------------------------------------------------------------------
+
+You can perform reads when ``canCommit`` is ``true``. However, the data that you 
+read is :term:`eventually consistent <eventual consistency>`, meaning that you 
+may not always read the latest writes.
+
+You can write to any non-synced namespaces while sync is in progress. If you 
```

Comment:
[question]

Would it be helpful to specify here:

> During an ongoing sync, you can write to any non-synced namespaces if `canWrite` is `true`.

This might make it clearer that writing while `canWrite` is `false` isn't really something they should be doing.